pb/config/config.proto

```diff
@@ -61,7 +61,7 @@ message TestGroup {
   // Path to the test result location, including protocol.
   // The location must be accessible from the Update Server, and not local.
   // GCS is usually used, making this field "gs://<bucket>"
-  string gcs_prefix = 2;
```

Comment:
So this breaks all open source usage of testgrid.

Reply:
Is this actively breaking things, or breaking upgrading other repos'/test-infra's upgrade?

Reply:
Ah nvm, answered in #103

Reply:
Someone doing a patch upgrade of their testgrid repo will hit this. IMO we should just patch it internally when/if we want to use a different name for this field.

Reply:
I'm not sure that's worth it long-term since gcs_prefix is a very specific name for the field anyways (though I'm fine with it short-term to get test-infra updated). Can I just change the usage of this in test-infra afterwards?

Reply:
(along with making a new minor-version change I assume, to avoid hitting this when patching?)